VoltAir/qml/actors/Orb.qml

```diff
@@ -59,16 +58,24 @@ Actor {
         cacheRenderParams: true
         cullingControlsBodyActive: true
         pauseWhenHidden: true
-        frameCount: 11
+        frameCount: 1 //11
```

Comment:
Can we change back to 11 frames and a transparent background 11 times taller? This would be required for an animated background, right?